src/libsystemd-network/sd-ipv4acd.c

```diff
@@ -71,14 +71,19 @@ struct sd_ipv4acd {
         IPv4ACDState state;
         int ifindex;
         int fd;
-        int iteration;
-        int conflict;
+
+        unsigned n_iteration;
+        unsigned n_conflict;
```

Comment:
I'm not sure if this rename is beneficial. We use "n_" prefix when we have a variable "somethings", and need a counter to interpret it "n_somethings". "Iteration" carries an inherent meaning of counting by itself.